blocks/library/audio/test/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import '..';
```

Comment:
Would `import '../';` (slash) be more consistent with the rest of the codebase?

Reply:
Yes, can do it :)